### PR TITLE
Save message item raise an exception when subject greater than 255

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34985

## Description
Save message item raise an exception when subject greater than 255. Cutting the subject and add prefix indicates this subject was cut.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
this is the debug version for customer verification, correct?
looks good.

**Comment by tallieber** [comment]
> this is the debug version for customer verification, correct?
> looks good.

Yup, just a draft PR until they will approve it works.

File: Packs/EWS/Integrations/EWSv2/EWSv2.py [hunk 1]
```diff
@@ -1204,6 +1204,16 @@ def parse_incident_from_item(item, is_fetch):
             except ErrorIrresolvableConflict:
                 time.sleep(0.5)
                 item.save()
+            except ValueError as e:
+                if len(item.subject) > 255:
+                    demisto.debug("Length of message subject is greater than 255, item.save could not handle it, "
+                                  "cutting the subject.")
+                    sub_subject = "Length of subject greater than 255 characters. " \
+                                  "Partially subject: {}".format(item.subject[:180])
```
Comment (bakatzir) [inline Packs/EWS/Integrations/EWSv2/EWSv2.py:1212]:
```suggestion
                    sub_subject = "Length of subject greater than 255 characters. " \
                                  "Partial subject: {}".format(item.subject[:180])
```

File: Packs/EWS/ReleaseNotes/1_8_9.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### EWS v2
+- Fix an issue where ***fetch-incidents*** command did not handle message subject with length greater than 255 characters.
```
Comment (amshamah419) [inline Packs/EWS/ReleaseNotes/1_8_9.md:4]:
```suggestion
- Fix an issue where the ***fetch-incidents*** command would not handle a message subject with a length greater than 255 characters. 
```

File: Packs/EWS/Integrations/EWSv2/EWSv2.py [hunk 3]
```diff
@@ -1204,6 +1204,16 @@ def parse_incident_from_item(item, is_fetch):
             except ErrorIrresolvableConflict:
                 time.sleep(0.5)
                 item.save()
+            except ValueError as e:
+                if len(item.subject) > 255:
```
Comment (reutshal) [inline Packs/EWS/Integrations/EWSv2/EWSv2.py:1208]: Can item.subject be None?
If so, I suggest 
```suggestion
                if item.subject and len(item.subject) > 255:
```

**Review by reutshal — APPROVED** [review]

**Pull request merged**